front/src/modules/pathfinding/helpers/getStepLocation.ts

```diff
@@ -1,19 +1,35 @@
-import type { PathItemLocation } from 'common/api/osrdEditoastApi';
+import type { OperationalPointReference, PathItemLocation } from 'common/api/osrdEditoastApi';
 import { mToMm } from 'utils/physics';
 
 const getStepLocation = (step: PathItemLocation): PathItemLocation => {
+  const trackReference: OperationalPointReference['track_reference'] =
+    !('track' in step) && step.track_reference && 'track_name' in step.track_reference
+      ? { track_id: undefined, track_name: step.track_reference.track_name }
```

Comment:
```suggestion
      ? {track_name: step.track_reference.track_name }
```
Here track is always undefined.

Reply:
Done